std_msgs/Header header

#state of GPS:
bool gps_active
bool gps_fixed

sensor_msgs/NavSatFix nav_sat_fix

float64 utc
string latitude_ns
string longitude_ew
int8 position
int8 satellite_used
float32 hdop
float32 altitude
float32 gps_heading
float32 speed_over_ground
int8 mode
float32 track
float64 gps_x
float64 gps_y
float64 gps_z
string utm_zone


